consensus/enclave/trusted/Cargo.lock

```diff
@@ -619,25 +640,37 @@ checksum = "7ffc5c5338469d4d3ea17d269fa8ea3512ad247247c30bd2df69e68309ed0a08"
 
 [[package]]
 name = "mbedtls"
-version = "0.5.1"
-source = "git+https://github.com/mobilecoinofficial/rust-mbedtls.git?tag=mc-0.3#8cac1fd1b9c521efb742a1014471e8563b2d6426"
+version = "0.8.1"
+source = "git+https://github.com/mobilecoinofficial/rust-mbedtls.git?rev=358564b57a2ae51d06419878b0f073a0603028cc#358564b57a2ae51d06419878b0f073a0603028cc"
 dependencies = [
  "bitflags",
  "byteorder",
  "cc",
- "genio",
+ "cfg-if 1.0.0",
+ "chrono",
+ "core_io",
  "mbedtls-sys-auto",
+ "rs-libc",
```

Comment:
This is going to cause symbol issues with the `sgx_tcalloc`---they're both exporting `malloc`, `free`, & friends as C symbols.